python/arcticdb/arctic.py

```diff
@@ -140,6 +147,17 @@ def __init__(self, uri: str, encoding_version: EncodingVersion = DEFAULT_ENCODIN
 
                 Example connection strings are `lmdb:///home/user/my_lmdb` or `lmdb:///home/user/my_lmdb?map_size=2GB`.
 
+
+            In-Memory
+            ----
+                The in-memory connection scheme has the form ``mem://<name>`, e.g. `mem://memory_db`.
```

Comment:
I wonder what the point of `<name>` is then? For now, all I can it doing is adding confusion. Shall we just have `mem://`?

Reply:
- [ ] remove name